test/modules/TestResultCoordinator/CountingReport.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using System.Collections.Generic;
+    
+    class CountingReport<T> : TestResultReportBase
```

Comment:
Can you please add comments on these types to explain what they are supposed to do?